kafl_fuzzer/technique/grimoire_mutations.py

```diff
@@ -89,10 +94,14 @@ def mutate_input_extension(generalized_input, func, grimoire_inference):
 
     rand_generalized = random_generalized(grimoire_inference)
 
-    data = grimoire_inference.generalized_to_string(rand_generalized) + grimoire_inference.generalized_to_string(generalized_input)
+    data = grimoire_inference.generalized_to_string(rand_generalized) + grimoire_inference.generalized_to_string(
+        generalized_input
```

Comment:
does it help to add a manual newline at '+' ?

I think we should fix readability issues like this before applying auto-format...